test/specs/reference/tests/14.pseudo_columns.js

```diff
@@ -474,10 +474,10 @@ exports.execute = function (options) {
 
             describe('when visible-columns annotation is not present for the context, ', function () {
                 describe('PseudoColumn for key, ', function () {
-                    it('if key is simple and its contituent columns are part of simple foreign key, should not be added (instead it should apply the PseudoColumn for foreignkey logic.)', function(done) {
+                    it('if key is simple and its constituent columns are part of simple foreign key, should not be added (instead it should apply the PseudoColumn for foreignkey logic.)', function(done) {
```

Comment:
Now that you have changed the schema, this test case and the one after are not testing what they're supposed to test. Just `xit` them and add a `TODO` for me please.

Reply:
Done.